shell/common/node_bindings.cc

```diff
@@ -151,6 +152,25 @@ void V8FatalErrorCallback(const char* location, const char* message) {
   *zero = 0;
 }
 
+bool AllowWasmCodeGenerationCallback(v8::Local<v8::Context> context,
+                                     v8::Local<v8::String>) {
+  // If we're running with contextIsolation enabled, fall back to Blink's logic.
+  // We switch on the embedder data field count here because if we're in a ctx
+  // that Node.js doesn't control, the number of embedder data fields will
+  // always be less than the min index Node.js uses.
+  if (context->GetNumberOfEmbedderDataFields() <
```

Comment:
We may want to instead switch this on `node::Environment::GetCurrent` being `nullptr` 🤔

Reply:
(gonna do that)

Reply:
Yeah I think that makes sense